src/__tests__/standardMethods.spec.ts

```diff
@@ -27,6 +30,26 @@ function numberGeneratorAPIEndpoint(config: AxiosRequestConfig) {
   }
 }
 
+function countChunks(axiosMock: MockAdapter, url: string) {
+  const counterObject = { value: 0 };
+  axiosMock.onPost(url).reply(config => {
+    counterObject.value++;
+    const { items: requestItems } = JSON.parse(config.data);
+    const responseItems = requestItems.map((item: any) => ({
+      ...item,
+      id: Number(item.name),
+    }));
+    return [201, { items: responseItems }];
+  });
+  return counterObject;
+}
+
+function fillArray() {
+  return new Array(2003).fill(null).map((_, index) => ({
```

Comment:
you don't need to fill it with nulls, as you map it right afterwards

Reply:
`.fill` requires a value...